components/topic/PrayerSummary.tsx

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import { Container, Card } from "react-bootstrap";
+import {FaPrayingHands} from 'react-icons/fa'
+import { IconContext } from "react-icons/lib";
+import { BsDownload } from "react-icons/bs";
+
+interface Props {
+    prayerPoints: string[]
+    title: string
+    style?: any
```

Comment:
I think we'll want to ask Design team to maybe make those components more similar, because theres still a bunch of minor style differences between the top and bottom prayer list component things. If they become more similar in design then I can work on making it reusable depending on where it is.

As for the style... I'm not sure why its there LOL and I don't think it's being used. I think I'll remove it.